src/FsToolkit.ErrorHandling.TaskResult/Task.fs

```diff
@@ -12,12 +12,19 @@ module Task =
       return! f x
   }
 
+  let bindV (f : 'a -> Task<'b>) (x : ValueTask<'a>) = task {
+      let! x = x
+      return! f x
+  }
+
   let apply f x =
     bind (fun f' ->
       bind (fun x' -> singleton(f' x')) x) f
 
   let map f x = x |> bind (f >> singleton)
 
+  let mapV f x = x |> bindV (f >> singleton)
+
```

Comment:
Not sure about the general utility of these 2 functions. If you prefer, I could remove them and just do

```fsharp
task {
    let! t = t
    return Ok t // and Some t
}
```

in the `Source` methods for `ValueTask<'t>`. They'd also run a tiny bit faster as a result.

Reply:
> They'd also run a tiny bit faster as a result.

Yeah I've been wondering if doing pure functions is better or not. If you could show this via benchmarks.net I'd highly appreciate it! 😄 


I'm going to accept this PR as the slight speedup isn't a blocker for getting this awesome functionality out the door.